### PR TITLE
Load environment texture from .babylon

### PR DESCRIPTION
**Pull request by noalak**
Also update world extends to ignore meshes at infinite distance like default skybox

### CODE REVIEW

File: src/Loading/Plugins/babylon.babylonFileLoader.ts [hunk 1]
```diff
@@ -529,6 +529,16 @@
                     }
                 }
 
+                // Environment texture
+                if (parsedData.environmentTexture !== undefined && parsedData.environmentTexture !== null) {
+                    scene.environmentTexture = Texture.Parse(parsedData.environmentTexture, scene, rootUrl);
```
Comment (deltakosh) [inline src/Loading/Plugins/babylon.babylonFileLoader.ts]: Don't we want to call CreateFromFilteredData here?

Comment (noalak) [reply src/Loading/Plugins/babylon.babylonFileLoader.ts]: When using CreateFromFilteredData in Sandbox, need to add "file:" before texture name.

**Comment by deltakosh** [comment]
Yes agree and this should work:)
We cannot change the way loader behaves just for the sandbox 
As all other textures work in sandbox, I'm sure we can fix the CreateFromFilteredData to detect when file: need to be added

**Comment by noalak** [comment]
I will have a look at it tomorrow

**Comment by deltakosh** [comment]
Thanks a lot

**Pull request closed without merging**